src/nucleus/nucleus.mli

```diff
@@ -475,15 +479,21 @@ val print_eq_type_abstraction :
   ?max_level:Level.t -> penv:print_environment -> eq_type_abstraction -> Format.formatter -> unit
 
 val print_judgement_abstraction :
-  ?max_level:Level.t -> penv:print_environment -> judgement_abstraction -> Format.formatter -> unit
+  ?max_level:Level.t -> penv:print_environment -> sgn:signature -> judgement_abstraction -> Format.formatter -> unit
```

Comment:
Reinstante the old signature. Printing routines should only require `penv` to print things.